internal/commands/rebase.go

```diff
@@ -49,6 +49,7 @@ func Rebase(logger logging.Logger, cfg config.Config, pack PackClient) *cobra.Co
 	cmd.Flags().BoolVar(&opts.Publish, "publish", false, "Publish to registry")
 	cmd.Flags().StringVar(&opts.RunImage, "run-image", "", "Run image to use for rebasing")
 	cmd.Flags().StringVar(&policy, "pull-policy", "", "Pull policy to use. Accepted values are always, never, and if-not-present. The default is always")
+	cmd.Flags().StringSliceVarP(&opts.Tags, "tag", "t", nil, "Comma-separated list of tag to apply to the rebased image")
```

Comment:
I would love for a maintainer to weigh in here (@jjbustamante @jkutner), but per [this comment](https://github.com/buildpacks/pack/issues/1748#issuecomment-1538742701) it was suggested to re-use the `--previous-image` flag from `pack build`. We could append something like `Use this flag in combination with <image-name> to avoid replacing the original image.` to the help text.

Reply:
@natalieparellano I confusion here , please do let me know if I am wrong. 
so the command goes like (rebase image `Image_name:v1` to `Image_name:v2`)- 
```
pack rebase --previous image=Image_name:v2 image_name:v1
```
or 
```
pack rebase --previous image=Image_name:v1 image_name:v2
```
I think the first one don't justifies the flag name ..

Reply:
okay I think the confusion was not valid indeed, I misunderstood the logic behind the rebasing an image, sorry about that.